newrelic/hooks/mlmodel_openai.py

```diff
@@ -29,6 +30,8 @@ def wrap_embedding_create(wrapped, instance, args, kwargs):
     if not transaction:
         return wrapped(*args, **kwargs)
 
+    transaction.add_ml_model_info("OpenAI", get_package_version("openai"))
```

Comment:
We need to move the `get_package_version("openai")` to the top of the module so that it doesn't get computed every time for performance reasons.

Reply:
Good catch!